tests/mitoai_ui_tests/mitoai.spec.ts

```diff
@@ -35,6 +35,21 @@ test.describe('Mito AI Chat', () => {
     expect(code?.trim()).toBe("")
   });
 
+  test('Edit Message', async ({ page }) => {
```

Comment:
Can we update this test to make sure that when you edit a message it clears all future messages from the chat history also? I think to do that, we should update the helper function you created from sending and immiedietly editing the message, to a function called `editMitoAIMesage` that takes a message index and a new message. 

That way we can do things like:
1. Create a first message 
2. Create a second message
3. Use the helper function to edit the first message